Trakt/Helpers/LibraryManagerEventsHelper.cs

```diff
@@ -95,103 +101,115 @@ private async void OnQueueTimerCallback(object state)
         private async Task OnQueueTimerCallbackInternal()
         {
             _logger.LogInformation("Timer elapsed - processing queued items");
+            var queue = new List<LibraryEvent>();
```

Comment:
unused

Reply:
I'll do a cleanup PR later to fix those things. Thanks for reviewing all the time.

Reply:
No, you immediately replace it `queue = _queuedEvents.ToList();`

Reply:
So I could either do `AddRange` within the lock or don't inititialize with an empty list? According to VS Code I can't just do `var queue = _queuedEvents.ToList();` within the lock.

Reply:
Don't initialize